table.csv

```diff
@@ -67,6 +67,7 @@ eth-account-snapshot,           ipld,           0x97,           permanent,  Ethe
 eth-storage-trie,               ipld,           0x98,           permanent,  Ethereum Contract Storage Trie (Eth-Secure-Trie)
 eth-receipt-log-trie,           ipld,           0x99,           draft,      Ethereum Transaction Receipt Log Trie (Eth-Trie)
 eth-receipt-log,                ipld,           0x9a,           draft,      Ethereum Transaction Receipt Log (RLP)
+eth-solidity-abi,               ipld,           0x9b,           draft,      Ethereum Solidity ABI-encoded data
```

Comment:
```suggestion
eth-solidity-abi,               ipld,           0x9b,           draft,      ABI-encoded contract calls from EVM environments
```

Reply:
not just "calls", returns can also be encoded in this form too

Reply:
none of this is correct.
(a) as mentioned above, Solidity ABI encoding is used for more than just calls.
(b) the EVM does not care about the ABI
(c) it is Ethereum-related, so let's keep Ethereum in the description